tasm-staff-editor/src/components/EditorView/ExhibitForm.js

```diff
@@ -1,11 +1,11 @@
 import React, { useState } from 'react';
-import QRCodeComponent from '../QRCode/QRCodeComponent.js';
+import QRCodeComponent from './QRCodeComponent.js';
 import { db } from '../../firebase.js';
 import { collection, addDoc, updateDoc } from 'firebase/firestore';
-import ExhibitTitle from '../ExhibitTitle/ExhibitTitle.js';
-import MediaType from '../MediaType/MediaType.js';
-import ExhibitContent from '../ExhibitContent/ExhibitContent.js';
-import ReadingLinks from '../ReadingLinks/ReadingLinks.js';
+import ExhibitTitle from './ExhibitTitle.js';
+import MediaType from './MediaType.js';
+import ExhibitContent from './ExhibitContent.js';
+import ReadingLinks from './ReadingLinks.js';
```

Comment:
love how moving the flies simplify these imports, might look at doing the same in the user-view down the road